src/wrapper-utils.js

```diff
@@ -74,81 +75,3 @@ export function wrapFunction (functionValue, realTarget) {
         }
     })
 }
-
-/**
- * Wrap a get/set or value property descriptor. Only for data properties. For methods, use wrapMethod(). For constructors, use wrapConstructor().
- * @param {any} object - object whose property we are wrapping (most commonly a prototype)
- * @param {string} propertyName
- * @param {Partial<PropertyDescriptor>} descriptor
- * @returns {PropertyDescriptor|undefined} original property descriptor, or undefined if it's not found
- */
-export function wrapProperty (object, propertyName, descriptor) {
-    if (!object) {
-        return
-    }
-    if (hasMozProxies) {
-        object = object.wrappedJSObject || object
-    }
-
-    const origDescriptor = getOwnPropertyDescriptor(object, propertyName)
-    if (!origDescriptor) {
-        // this happens if the property is not implemented in the browser
-        return
-    }
-
-    if (('value' in origDescriptor && 'value' in descriptor) ||
-        ('get' in origDescriptor && 'get' in descriptor) ||
-        ('set' in origDescriptor && 'set' in descriptor)
-    ) {
-        wrapToString(descriptor.value, origDescriptor.value)
-        wrapToString(descriptor.get, origDescriptor.get)
-        wrapToString(descriptor.set, origDescriptor.set)
-
-        defineProperty(object, propertyName, {
-            ...origDescriptor,
-            ...descriptor
-        })
-        return origDescriptor
-    } else {
-        // if the property is defined with get/set it must be wrapped with a get/set. If it's defined with a `value`, it must be wrapped with a `value`
-        throw new Error(`Property descriptor for ${propertyName} may only include the following keys: ${objectKeys(origDescriptor)}`)
-    }
-}
-
-/**
- * Wrap a method descriptor. Only for function properties. For data properties, use wrapProperty(). For constructors, use wrapConstructor().
- * @param {any} object - object whose property we are wrapping (most commonly a prototype)
- * @param {string} propertyName
- * @param {(originalFn, ...args) => any } wrapperFn - wrapper function receives the original function as the first argument
- * @returns {PropertyDescriptor|undefined} original property descriptor, or undefined if it's not found
- */
-export function wrapMethod (object, propertyName, wrapperFn) {
```

Comment:
I'm assuming these have had as close to near zero changes to move into ContentFeature or should I be checking much more closely they're like for like?

Reply:
Yup, no changes except the added addDebugFlag() call